rules/Php73/Rector/FuncCall/RegexDashEscapeRector.php

```diff
@@ -42,7 +42,7 @@ final class RegexDashEscapeRector extends AbstractRector implements MinPhpVersio
      * @var string
      * @see https://regex101.com/r/TBVme9/8
```

Comment:
please update the regex101 uri as well to show how it differentiate it, after change, click "Update" instead of "Save" so we have continuity increment number `/9` of path.

Reply:
@samsonasik done https://github.com/rectorphp/rector-src/pull/2753/commits/6b223d1d09e7648a63d73925281b7508630c331c

Reply:
Could you update the regex101 content too for the failing test case? Or it already covered?

Reply:
It's already kinda covered by `[a\-\w]` not being matched its essentially the same thing. Honestly the slashes test case can be removed entirely in my opinion as it is the same as the test case I added at this point. Again this worries me because I didn't see the original use case from the issue as it was a broken link but the test case added from the issue I believe was incorrect from looking at regex101.

Reply:
Thank you, /cc @sidux could you verify it as the author that create original issue https://github.com/rectorphp/rector/issues/3190 for it? Thank you.